app/assets/javascripts/student_profile/page_container.js

```diff
@@ -54,6 +54,10 @@
 
         // ui
         selectedColumnKey: queryParams.column || 'interventions',
+        
+        // This holds the state of network requests, so that UI components can show that to the user.
+        // These might be null (no requests), 'pending' (request currently in-flight), and 'error' or another
+        // value if the request failed.
         requests: {
```

Comment:
This is good. The value of saveNotes and discontinueService will be JSON objects where the key is the note or service id and the value is null or a string?